ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -235,8 +257,13 @@ def test_run_ipahealthcheck_list_source(self):
         """
         Testcase to verify sources available in healthcheck tool.
         """
+        version = tasks.get_healthcheck_version(self.master)
         result = self.master.run_command(["ipa-healthcheck", "--list-sources"])
-        for source in sources:
+        if parse_version(version) > parse_version("0.4"):
```

Comment:
To be more accurate, the check should be:
`if parse_version(version) >= parse_version("0.6"):`
because 0.5 provides the same checks as 0.4.